src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -627,6 +627,21 @@ export default class ResultsViewPage extends React.Component<
                             />
                         </div>
                     )}
+
+                    {this.userMessages.isComplete &&
+                        this.userMessages.result.length > 0 && (
+                            <Portal
```

Comment:
would be nice if we could put portal business inside of the UserMessager component.